src/service.rs

```diff
@@ -45,10 +51,13 @@ pub struct Service<S> {
     config_service: ConfigService,
 
     // このサーバが所有するデバイス一覧
-    local_devices: HashMap<u32, LocalDevice>,
+    local_devices: HashMap<DeviceNo, LocalDevice>,
 
     // クラスタ全体の情報
-    seqno_to_device: HashMap<u32, PhysicalDevice>,
+    seqno_to_device: HashMap<DeviceNo, PhysicalDevice>,
+
+    // このサーバーに紐付いているノード一覧
+    local_nodes: HashMap<(BucketId, u16, entity::device::DeviceId), NodeId>,
```

Comment:
The purpose of the changes in this file is to get a mapping from a device to `NodeId`.